app/models/item_order.rb

```diff
@@ -1,4 +1,5 @@
 class ItemOrder < ActiveRecord::Base
+  validates_numericality_of :quantity, greater_than: 0
```

Comment:
Prefer the new style validations `validates :column, numericality: value` over `validates_numericality_of`.